src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java

```diff
@@ -95,6 +99,22 @@ public void setGameType(GameType gameType) {
     @Override
     public GameType getGameType(){return this.game.getGameType();}
 
+    private String generateInvitationCode() {
+        //set limits for including only alphanumeric values
+        int lowerLimit = 48;
+        int upperLimit = 123;
+
+        //set limit for the string length
+        int lengthLimit = 10;
+
+        Random random = new Random();
+        return random.ints(lowerLimit, upperLimit)
```

Comment:
I don't understand what you are completely doing, so maybe it is obvious, how do you check that the invitation code is unique over all lobbies ?

Reply:
When joining the lobby, we are sending the id of the lobby (unique), and the code, which is going to be matched with the one stored in the DB. So I think that the code does not need to be unique.

Reply:
Ahh, okey I interpreted the code as an additional way to identify a lobby, and you more like a password if I understand correctly? 
This naturally also works!

Reply:
nono, that's probably better.  Two different ways to identify a lobby are enough 😅